src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -513,4 +541,8 @@ public long getLong(String name) {
     public String[] getStringArray(String name) {
         return mProperties.getStringArray(name);
     }
+
+	public String getThriftProtocolClass() {
+        return mProperties.getString("secor.thrift.protocol.class");
```

Comment:
New property needs to be defined in secor.common.properties with documentation and default value.